data-processing/common/parse_equation.py

```diff
@@ -533,21 +597,39 @@ def _extract_tokens(element: Tag) -> List[Token]:
                 start=int(element["s2:start"]),
                 end=int(element["s2:end"]),
                 type_="atom",
+                mathml=str(cleaned_element),
+                font_macros=_extract_font_macros(element),
             )
         )
     elif _is_affix_token(element):
+        cleaned_element = clone_element(element)
+        _sanitize_attributes(cleaned_element)
         tokens.append(
             Token(
                 text=str(element.string),
                 start=int(element["s2:start"]),
                 end=int(element["s2:end"]),
                 type_="affix",
+                mathml=str(cleaned_element),
+                font_macros=_extract_font_macros(element),
             )
         )
 
     return tokens
 
 
+def _extract_font_macros(element: Tag) -> Tuple[str, ...]:
+    """
+    Get the list of TeX styling macros applied to a token represented by a MathML element. Relies
+    on the MathML having an "s2:font-macros" attribute on the elements for tokens, where
+    each font macro is separated by an '&' character.
```

Comment:
Kyle asks if we can add a pointer to the place in the KaTeX code that creates the font-macros property.